Rota-Process.md

```diff
@@ -0,0 +1,302 @@
+
+## Kata Review Team
+
+This Kata Containers project runs a [rolling rota of members and volunteers](#the-rota)
+who donate some of their time to try and assess, review and generally push
+along and reduce any PR and issue backlogs across the repositories.
+
+Informally, this team is sometimes referred to as the "Kat Herding Team".
+
+## The rota
+
+The rota is maintained in the
+[community wiki](https://github.com/kata-containers/community/wiki/Review-Team-Rota)
+since it is updated frequently.
+
+## Who can join?
+
+Anybody can participate. You do **not** have to be a subject matter expert or Kata
+expert to help! All contributions are welcome.
+
+Joining the team is a great way to expand your knowledge of the project and
+your skill set. Sometimes all it needs is for somebody to read the list of
+open PRs, nudge them along and ask the relevant subject matter experts if they
+can review or help with a PR.
+
+Note that you must agree to our [code of [conduct](CODE_OF_CONDUCT.md)
+if you wish to become involved.
+
+## How can I participate?
+
+If you wish to help out and want to be added to [the rota](#the-rota), please
+notify the Kata team via one of the contact methods listed on
+https://katacontainers.io before editing this page. Don't forget to include
+your contact details in the Rota table, preferably your GitHub name.
+
+## Preparation
+
+Before joining the team, it is strongly recommended that you read the
+[contribution guide](https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md).
+You may also wish to read the
+[PR review guide](https://github.com/kata-containers/community/blob/master/PR-Review-Guide.md)
+for tips on areas to think about when reviewing [pull requests](#prs).
+
+### Reviewing documentation changes
+
+You may wish to read the 
+[documentation requirements](https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md)
+and [documentation review](https://github.com/kata-containers/community/blob/master/Documentation-Review-Process.md)
+documents to help reviewing documentation changes.
+
+## Tasks
+
+This section lists the sorts of activities the team should try to focus on.
+
+### Talk to each other
+
+We suggest that those on the review team hang out on
+[Slack or IRC](README.md#join-us)
+so they can chat to each other. Remember, nobody knows everything so ask the
+other folk on the team if you need help!
+
+### PRs
+
+#### Review PR backlog
+
+See [the GitHub reports](https://github.com/kata-containers/community/wiki/Review-Team-Rota#finding-the-open-prs)
+for listing sets of open PRs.
+
+To learn how to "ack" a PR, read
+[the reviews section in the contributing guide](https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#reviews).
+
+##### Triggering a CI run
+
+If you have reviewed the PR and it looks sensible, you can trigger a CI run by
+adding a magic comment:
+
+```
+/test
+```
+
+If the CI needs to be re-run:
+
+```
+/retest
+```
+
```

Comment:
I'll hold off further updates here until we get more info.

Reply:
In fact, it might be best if this doc points to a wiki page that summarises the commands. That way the doc can remain relatively static but the wiki could change?

Reply:
yep, that works for me, and shifts it off your plate as well ;-)

Reply:
OK. PR updated with a link to the new wiki page:

- https://github.com/kata-containers/community/wiki/Controlling-the-CI

Reply:
Could folk please update ^ with all the remaining "magic commands"? /cc @chavafg, @grahamwhaley.

Reply:
One way to find those is going to be extract the `triggerPhrase` entries from the Jenkins job xml stored over in the CI repo, like: https://github.com/kata-containers/ci/blob/master/jenkins/jobs/kata-containers-agent-Power8-ubuntu-16-04-PR-initrd/config.xml#L73

Let me have a quick search...

Reply:
OK, a bit of grep/sed/sort/uniq comes up with:

```
<triggerPhrase>.*(\n|^|\s)/(re)?test(-arm)?(\n|$|\s)+.*</triggerPhrase>
<triggerPhrase>.*(\n|^|\s)/(re)?test(-fedora)?(\n|$|\s)+.*</triggerPhrase>
<triggerPhrase>*(\n|^|\s)/(re)?test(\n|$|\s)+.*</triggerPhrase>
<triggerPhrase>.*(\n|^|\s)/(re)?test(\n|$|\s)+.*</triggerPhrase>
<triggerPhrase>.*(\n|^|\s)/(re)?test(-nemu)?(\n|$|\s)+.*</triggerPhrase>
```
but, I'm pretty sure we might have one or two more than that - maybe the xml needs an update cycle. I'll slide those into the wiki page to start with and let @chavafg fix etc.